tests/PSDocs.Azure.Tests/template-test/template.json

```diff
@@ -2,7 +2,6 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "metadata": {
-        "name": "Storage Account",
```

Comment:
Did you want to show that this property will be overriden by "itemDisplayName" in metadata.json?

Reply:
Properties in the template will override metadata.json. Updated to make this clearer.